app/proxy/client.go

```diff
@@ -106,3 +111,13 @@ func (c *Client) isWalletNotLoaded(r *jsonrpc.RPCResponse) bool {
 	}
 	return false
 }
+
+func (c *Client) isWalletAlreadyLoaded(r *jsonrpc.RPCResponse) bool {
+	if r.Error != nil {
+		wErr := lbrynet.NewWalletError(0, errors.New(r.Error.Message))
+		if errors.As(wErr, &lbrynet.WalletAlreadyLoaded{}) {
```

Comment:
shouldn't this be `errors.Is`? i thought `errors.As` is for when you want to cast the error for use later

Reply:
The way it's named is not exactly intuitive. Here's gist of how it works:

> In the simplest case, the errors.Is function behaves like a comparison to a sentinel error, and the errors.As function behaves like a type assertion.